scripts/run/gce/logger_daemon.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+
+# Add the following to jbox.user
+# 'cloud_host' : {
+#     ...,
+#     'log-interval': 5, # seconds
+# }
+
+
+from time import sleep, time
+import os
+from sys import argv
+import httplib
+from socket import gethostname
+from oauth2client.client import GoogleCredentials
+from googleapiclient.discovery import build
+import errno
+
+if len(argv) < 2:
+    print('USAGE: ./logger_daemon.py <log_files_paths>')
+    exit(-1)
+
+conf = None
+with open('/jboxengine/conf/jbox.user') as f:
+    conf = eval(f.read())
+LOGPATHS = argv[1:]
+ch = conf['cloud_host']
+INTERVAL = ch['log-interval']
+creds = GoogleCredentials.get_application_default()
+conn = build('logging', 'v2beta1', credentials=creds)
+MYNAME = gethostname()
+logurl = 'projects/%s/logs/%s' % (ch['install_id'], MYNAME)
+
+def retry_on_bsl(f):
+    def func(*args, **kwargs):
+        try:
+            return f(*args, **kwargs)
+        except httplib.BadStatusLine as bsl:  # Retry on bad status line
+            return f(*args, **kwargs)
+        except IOError as e:
+            if e.errno == errno.EPIPE:        # Retry on broken pipe errno 32
+                return f(*args, **kwargs)
+            else:
+                raise
+    return func
+
+@retry_on_bsl
+def cloud_log(line, sev=None):
+    if not sev:
+        try:
+            sev = line.split(' - ')[1]
+            if sev not in ['INFO', 'DEBUG', 'WARNING', 'CRITICAL', 'ERROR']:
+                sev = 'INFO'
+        except:
+            sev = 'INFO'
+
+    ent = conn.entries()
+    body = {
+        'entries': [
+            {
+                'severity': sev,
+                'textPayload': line,
+                'logName': logurl,
+                'resource': {
+                    'labels': {
+                        'instance_id': 'testbox',
+                        'zone': 'us-central1-c'
```

Comment:
Should `instance_id` and `zone` have hard-coded values?
It will be better if they can be determined from configuration or environment or an api call. That will help in subsequent troubleshooting/analysis.